### PR TITLE
Sync User Properties

### PR DESCRIPTION
**Pull request by TheConnMan**

### CODE REVIEW

**Comment by TheConnMan** [comment]
Ref #185

File: assets/js/profile.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      });
+      persistProperties();
+    });
+  }
+
+  function persistProperty(key, value) {
+    $storage.set(key, value);
+    var payload = {};
+    payload[key] = value;
+    return loggedIn ? $http.post('/profile/add', payload) : Promise.resolve();
```
Comment (TheConnMan) [inline assets/js/profile.js]: I hadn't intended to use sockets exclusively. Videos and chats are global so it makes sense to be sockets, but user properties are per user, so no need to broadcast.

Comment (SirZach) [inline assets/js/profile.js]: it's not broadcast going from client-to-server

Comment (TheConnMan) [inline assets/js/profile.js]: I meant broadcast user property changes from server to client which would encourage use of websockets for communication both ways.

Comment (SirZach) [inline assets/js/profile.js]: like i said originally, it makes sense to me as an `http` request so lg2m

**Pull request merged**